day_two_guide/topics/scaling_etcd.adoc

```diff
@@ -0,0 +1,432 @@
+status by moving the new etcd host from the `[new_etcd]` group to the `[etcd]`
+group:
++
+----
+[OSEv3:children]
+masters
+nodes
+etcd
+<new_etcd>
+
+... [OUTPUT ABBREVIATED] ...
+
+[etcd]
+master-0.example.com
+master-1.example.com
+master-2.example.com
+etcd0.example.com
+----
+
+. If using Flannel, modify the `flanneld` service configuration, located at
+`/etc/sysconfig/flanneld` on every {product-title} host, to include the new etcd
+host:
++
+----
+FLANNEL_ETCD_ENDPOINTS=https://master-0.example.com:2379,https://master-1.example.com:2379,https://master-2.example.com:2379,https://etcd0.example.com:2379
+----
+
+. Restart the `flanneld` service:
++
+----
+# systemctl restart flanneld.service
+----
+
+==== Manually adding a new etcd host
+
+The following steps can be performed on any etcd member. If using the Ansible
+installer, the first host provided in the `[etcd]` Ansible inventory is used to
+generate the etcd configuration and certificates stored in
+`/etc/etcd/generated_certs`, so perform the next steps in that etcd host.
+
+*Steps to be performed on the current etcd cluster*
+
+[discrete]
+== Procedure
+
+. In order to create the etcd certificates, run the `openssl` command with the
+proper values. To make this process easier, create some environment variables:
++
+----
+export NEW_ETCD_HOSTNAME="*etcd0.example.com*"
+export NEW_ETCD_IP="192.168.55.21"
+
+export CN=$NEW_ETCD_HOSTNAME
+export SAN="IP:${NEW_ETCD_IP}"
+export PREFIX="/etc/etcd/generated_certs/etcd-$CN/"
+export OPENSSLCFG="/etc/etcd/ca/openssl.cnf"
+----
++
+[NOTE]
+====
+The custom `openssl` extensions used as `etcd_v3_ca_*` include the
+$SAN environment variable as `subjectAltName`. See `/etc/etcd/ca/openssl.cnf`
+for more information.
+====
+
+. Create the directory where the configuration and certificates are stored:
++
+----
+# mkdir -p ${PREFIX}
+----
+
+. Create the server certificate request and sign it:
++
+----
+# openssl req -new -config ${OPENSSLCFG} \
+    -keyout ${PREFIX}server.key  \
+    -out ${PREFIX}server.csr \
+    -reqexts etcd_v3_req -batch -nodes \
+    -subj /CN=$CN
+
+# openssl ca -name etcd_ca -config ${OPENSSLCFG} \
+    -out ${PREFIX}server.crt \
+    -in ${PREFIX}server.csr \
+    -extensions etcd_v3_ca_server -batch
+----
+
+. Create the peer certificate request and sign it:
++
+----
+# openssl req -new -config ${OPENSSLCFG} \
+    -keyout ${PREFIX}peer.key \
+    -out ${PREFIX}peer.csr \
+    -reqexts etcd_v3_req -batch -nodes \
+    -subj /CN=$CN
+
+# openssl ca -name etcd_ca -config ${OPENSSLCFG} \
+  -out ${PREFIX}peer.crt \
+  -in ${PREFIX}peer.csr \
+  -extensions etcd_v3_ca_peer -batch
+----
+
+. Copy the current etcd configuration and `ca.crt` files from the current node
+as examples to be modified later:
++
+----
+# cp /etc/etcd/etcd.conf ${PREFIX}
+# cp /etc/etcd/ca.crt ${PREFIX}
+----
+
+. Add the new host to the etcd cluster. Note the new host is not configured yet
```

Comment:
Would the bit about the status make more sense somewhere else? I don't see where that's displayed in the output.